types/ckeditor__ckeditor5-easy-image/ckeditor__ckeditor5-easy-image-tests.ts

```diff
@@ -1,42 +1,14 @@
 import { Editor } from "@ckeditor/ckeditor5-core";
-import { ExportWord } from "@ckeditor/ckeditor5-export-word";
-import { ExportWordConfig } from "@ckeditor/ckeditor5-export-word/src/exportword";
```

Comment:
Any reason to remove these tests? Have you moved them somewhere else?

Reply:
These tests appear to have been copied from the export to pdf plugin (where they still reside)...I removed them from here because these are the easy-image tests